application/modules/setup/sql/029_1.5.6.sql

```diff
@@ -0,0 +1,3 @@
+#Fix for IP-623
```

Comment:
You can remove that as it will be automatically added if the settings are updated :)